base/steer/FairRootManager.cxx

```diff
@@ -296,12 +491,20 @@ TClonesArray* FairRootManager::Register(TString branchName, TString className, T
     outputArray = fActiveContainer[branchName];
     Register(branchName, folderName, outputArray, toFile);
   }
+
+  LogMessage(TString("Done Register (3)") + branchName);
+#if ( ROOT_VERSION_CODE >= ROOT_VERSION(6,9,3) )
```

Comment:
If I understand correctly, if ROOT version is below of what you check here, the code will have races.

Feel free to bump the required version of ROOT in the main CMakeLists.txt to 6.10.0:
```
find_package(ROOT 6.00.00 REQUIRED) 
```
to

```
find_package(ROOT 6.10.00 REQUIRED) 
```
Then you can remove all the version checks in the code.

Reply:
Ok, I was not sure, if you are ok to set the minimal version to 6.10.00 globally.
The ifs inside code would keep the class compiling with older Root versions, but it would not be usable with MT. Do you want I commit this change in the branch?